models/surveys.go

```diff
@@ -29,53 +126,195 @@ func AllSurveys() ([]*SurveySummary, error) {
 		err := rows.Scan(&surveySummary.ID, &surveySummary.ShortName)
 
 		if err != nil {
-			return nil, err
+			http.Error(w, "No surveys found", http.StatusInternalServerError)
```

Comment:
Should we return after this to retain the http 500?

Reply:
good spot - fixed